apps/dashboard/src/utils/segment.ts

```diff
@@ -0,0 +1,135 @@
+import { AnalyticsBrowser } from "@segment/analytics-next";
+import { IUserEntity } from "@novu/shared";
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** we need to discuss whether we want to use the `@novu/shared` package, because it's a kind of bloated package

Reply:
https://novu.slack.com/archives/C07HVLG7G2U/p1727277810634729

Reply:
we decided to use `import type ... from '@novu/shared';`

Reply:
Can we add a quick eslint rule for this?